routes/auth.ts

```diff
@@ -4,11 +4,16 @@ import auth from "../controllers/auth";
 import authenticate from "../middlewares/authenticate";
 import userDeviceInfoValidator from "../middlewares/validators/qrCodeAuth";
 import qrCodeAuthValidator from "../middlewares/validators/qrCodeAuth";
+import { devFlagMiddleware } from "../middlewares/devFlag";
 
 router.get("/github/login", auth.githubAuthLogin);
 
 router.get("/github/callback", auth.githubAuthCallback);
 
+router.get("/google/login", devFlagMiddleware, auth.googleAuthLogin);
+
+router.get("/google/callback", auth.googleAuthCallback);
```

Comment:
should this also be behind dev flag?

Reply:
When Google redirects to the callback URL, it usually appends its own parameters (such as code, state, etc.) but will not include any query parameters like dev=true. and if a user directly visits this url he will not be able to authenticate and receive 401 error.